docs/api/sql/Raster-operators.md

```diff
@@ -172,6 +172,46 @@ Output:
 -2
 ```
 
+### RS_SkewX
+
+Introduction: Returns the X skew or rotation parameter.
+
+Format: `RS_SkewX(raster: Raster)`
+
+Since: `v1.5.0`
+
+Spark SQL Exmaple:
+
+```sql
+SELECT RS_SkewX(raster) FROM rasters
```

Comment:
Can you use MakeEmptyRaster as the input?

Reply:
Sure.